lib/chelsea/iq_client.rb

```diff
@@ -67,27 +69,47 @@ def poll_status(url)
           res = _poll_iq_server(url)
           if res.code == 200
             spin.success("...done.")
-            _handle_response(res)
-            break
+            return _handle_response(res)
           end
         rescue
           sleep(1)
         end
       end
     end
 
+    # colors to use when printing message
+    COLOR_FAILURE = 31
+    COLOR_WARNING = 30 # want yellow, but doesn't appear to print
```

Comment:
https://www.lihaoyi.com/post/BuildyourownCommandLinewithANSIescapecodes.html#8-colors

I can't remember for sure but one of the libraries we were using (maybe it's bach) was limited to 8 bit terminal support or something akin. 

ALSO, you may like this (because it's a nifty pastel thing):

`pastel.alias_color(:failure, :red, :bold)`

That sets a method alias so you can call `pastel.failure` and it'll print with the colors you picked for it.

Reply:
Interesting. Might revisit, but I'm gonna punt on the color stuff for now (itching to add RuboCop).